app/views/admin/pages/index.html.erb

```diff
@@ -2,14 +2,50 @@
 
 <header class="flex items-center mb-6">
   <h2 class="crayons-title">Pages</h2>
-  <%= link_to "New page", new_admin_page_path, class: "btn btn-primary ml-auto" %>
+  <%= link_to "New page", new_admin_page_path, class: "ml-auto crayons-btn crayons-btn--s" %>
 </header>
 
-<div class="crayons-card p-6">
+<div class="crayons-card p-6 pages__table">
   <% @pages.each do |page| %>
     <div class="flex py-2 items-center">
-      <%= link_to page.title, edit_admin_page_path(page.id) %>
-      <%= link_to "View", page.path, class: "ml-auto btn btn-secondary" %>
+      <%= link_to page.title, page.path %>
+      <%= link_to "Edit", edit_admin_page_path(page.id), class: "ml-auto crayons-btn crayons-btn--s crayons-btn--secondary" %>
     </div>
   <% end %>
 </div>
+
+<div class="mt-8 pages__override_defaults">
+  <div class="my-6">
+    <% if !@code_of_conduct || !@privacy || !@terms%>
```

Comment:
We show a warning that the user should proceed with caution when overriding any of the sections below

Reply:
**Suggestion**: could extract this into a helper that checks for presence of these pages